InternTrack.Core.Api/Models/Interns/Exceptions/NotFoundInternException.cs

```diff
@@ -13,5 +13,9 @@ public class NotFoundInternException : Xeption
         public NotFoundInternException(Guid internId)
             : base(message: $"Couldn't find intern id: {internId}.")
```

Comment:
If accepted please correct in the test as well.

```suggestion
            : base(message: $"Intern with id: {internId} not found, please correct and try again.")
```